### PR TITLE
Fix 1651: make word_vec() return immutable vector

### PR DESCRIPTION
**Pull request by CLearERR**
fix for issue about mutable vector returned by KeyedVectors.word_vector

### CODE REVIEW

File: gensim/models/keyedvectors.py [hunk 1]
```diff
@@ -278,11 +278,12 @@ def word_vec(self, word, use_norm=False):
         """
         if word in self.vocab:
             if use_norm:
-                return self.syn0norm[self.vocab[word].index]
+                result = self.syn0norm[self.vocab[word].index]
             else:
-                return self.syn0[self.vocab[word].index]
-        else:
```
Comment (menshikh-iv) [inline gensim/models/keyedvectors.py]: Why you remove this branch? We must raise an exception if word not in the dictionary.

File: gensim/models/keyedvectors.py [hunk 2]
```diff
@@ -278,11 +278,12 @@ def word_vec(self, word, use_norm=False):
...
-            raise KeyError("word '%s' not in vocabulary" % word)
+                result = self.syn0[self.vocab[word].index]
+
+            result.setflags(write=False)
```
Comment (menshikh-iv) [inline gensim/models/keyedvectors.py]: Please add test to check this behaviour (with Exception catching)

**Comment by gojomo** [comment]
Ideally PR names should be descriptive even if issue-numbers (like `#1651`) are ignored/unknown. So I've updated the name to be more descriptive.

**Comment by piskvorky** [comment]
On github, PR titles do not link through to the issues (no link in issue, no easy to way to visit the 
 issue).

Please use native Github formatting, it makes maintenance easier.

**Comment by menshikh-iv** [comment]
Thanks @CLearERR, congratz with the first PR :1st_place_medal:

**Pull request merged**